### ISSUE TITLE
Update Readme

### ISSUE THREAD

**Issue by SatyamKumarr** [chetanya-shrimali/nutrient-management-website#25]
Update your README.md to make it more user friendly. State the objective of your project and later state how to install using git clone.

**Comment by chetanya-shrimali** [comment]
Can you do that?

**Comment by SatyamK23** [comment]
Yeah sure.What extra feature have you include in your case study? Which features have you excluded from case study?

**Comment by chetanya-shrimali** [comment]
Features are
- Posting article
- ADD/edit/delete of particular article by particular user
- Like/comment feature for each article
- Created API's for the data stored
- Nutrients-info 
    - based on avg_intake, disease by lack or surplus and foods to prevent it
    - foods available in different regions and geographical areas
    - nutrients for different type of people
- Drop a note feature to get feedbacks and display them along with email of thanks send to the user.

**Comment by chetanya-shrimali** [comment]
organize it well and send the pr!!
